src/app/lib/utilities/getToggles/withCache.js

```diff
@@ -1,7 +1,11 @@
 import Cache from 'lru-cache';
 import getToggles from '.';
 
-const cacheMaxItems = parseInt(process.env.SIMORGH_CONFIG_CACHE_ITEMS, 10);
+const CACHE_MAX_ITEMS_FALLBACK = 100;
+const cacheMaxItems = parseInt(
+  process.env.SIMORGH_CONFIG_CACHE_ITEMS || CACHE_MAX_ITEMS_FALLBACK,
```

Comment:
`process.env.SIMORGH_CONFIG_CACHE_ITEMS` is defined for all environments. I'm not sure why we now need a fallback

Reply:
The test below was failing because `process.env.SIMORGH_CONFIG_CACHE_ITEMS` is undefined. I am assumed that this was caused because the test does not know in which environment it is, therefore the value will be undefined. Not sure how to force an environment for a test.

<img width="1396" alt="Screenshot 2022-03-25 at 10 22 27" src="https://user-images.githubusercontent.com/90621252/160102889-ee2e221a-1a52-4755-81a7-4c30dfd13301.png">

Reply:
ah ok. You can just mock that in the test e.g.

`process.env.SIMORGH_CONFIG_CACHE_ITEMS = 100`

Reply:
The problem was that withCache.js runs on import, hence I can not mock it.
I think I fixed this by requiring the module after this [line](https://github.com/bbc/simorgh/blob/3a90e58576899849b1c94de1b30558c2ddf832b1/src/server/index.test.jsx#L18).